test-runner/src/main/scala/com/typesafe/spark/test/mesos/DCOSSpec.scala

```diff
@@ -0,0 +1,22 @@
+package com.typesafe.spark.test.mesos
+
+import org.scalatest.FunSuite
+import org.scalatest.concurrent.TimeLimitedTests
+
+import com.typesafe.spark.test.mesos.framework.runners.RoleConfigInfo
+
+class DCOSSpec (
```

Comment:
This removes DynamicAllocation from the DCOS tests.  I think we can probably do better in the future so that new tests are auto added to this Spec, possibly by having a single Spec, and tagging individual tests as @ignore(dcos) or something.  But this works for now.